BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/ReturnAssistantConfiguration.swift

```diff
@@ -22,53 +22,6 @@ public final class ReturnAssistantConfiguration: NSObject {
      - returns: Instance of `ReturnAssistantConfiguration`.
      */
     public override init() {}
-    // MARK: Return Assistant
```

Comment:
We need to revert the changes in [ReturnAssistantConfiguration] 😊

Reply:
We can revert it, but these are not used anywhere in the app, other parts of the ReturnAssistantConfiguration is untouched which is used.